railties/test/commands/devcontainer_test.rb

```diff
@@ -8,33 +8,39 @@ class Rails::Command::DevcontainerTest < ActiveSupport::TestCase
 
   teardown { teardown_app }
 
-  test "generates devcontainer for default app" do
+  test "generates devcontainer for default app with solid gems" do
     build_app
 
+    require "solid_cable"
+    require "solid_queue"
+
     output = rails "devcontainer"
 
     assert_match "app_name: app_template", output
     assert_match "database: sqlite3", output
     assert_match "active_storage: true", output
-    assert_match "redis: true", output
+    assert_match "redis: false", output
     assert_match "system_test: true", output
     assert_match "node: false", output
   end
 
-  test "generates devcontainer for using mysql2 app" do
+  test "generates dev container for without solid gems" do
+    build_app
+
+    output = rails "devcontainer"
+
+    assert_match "redis: true", output
+  end
+
+  test "generates dev container for using mysql2 app" do
     build_app
 
     Dir.chdir(app_path) do
       use_mysql2
 
       output = rails "devcontainer"
 
-      assert_match "app_name: app_template", output
       assert_match "database: mysql", output
```

Comment:
Make this test a little less noisy since this is what we really care about. Then it is less sensitive to unrelated changes to how we generate dev containers.